### PR TITLE
Correct heron developers doc.

### PR DESCRIPTION
**Pull request by maosongfu**
1. Correct out-dated docs.
2. Fix broken links.

### CODE REVIEW

**Comment by lucperkins** [comment]
@maosongfu As you can see from the [current version of this doc](http://twitter.github.io/heron/docs/contributors/custom-metrics-sink/), the API versions aren't being properly inserted into the doc. This was an oversight on my part. I'll make this work properly in another PR tomorrow. Make sure that those changes get pushed before you push these changes.

**Comment by maosongfu** [comment]
@lucperkins sure.

File: website/content/docs/contributors/custom-scheduler.md [hunk 1]
```diff
@@ -3,39 +3,39 @@ title: Implementing a Custom Scheduler
 ---
 
 To run a Heron cluster, you'll need to set up a scheduler that is responsible
-for cluster management. Heron currently supports two schedulers out of the box:
+for cluster management. Heron currently supports followings schedulers out of the box:
```
Comment (billonahill) [inline website/content/docs/contributors/custom-scheduler.md]: s/followings/the following

File: website/content/docs/contributors/custom-scheduler.md [hunk 2]
```diff
@@ -46,48 +46,26 @@ interfaces:
...
-[`DefaultConfigLoader`](/api/scheduler/com/twitter/heron/scheduler/util/DefaultConfigLoader.html)
-interface.
-
-## Configurable Parameters
-
-At the very least, your configuration loader will need to be able to load the
-class names (as strings) for your implementations of the components listed
-above, as you can see from the interface definition for
-[`IConfigLoader`](/api/com/twitter/heron/spi/scheduler/IConfigLoader.html).
+Those interfaces and implementation are independent; you can choose and combine them to fit your use scenarios.
```
Comment (billonahill) [inline website/content/docs/contributors/custom-scheduler.md]: s/Those interfaces/The interfaces listed above

File: website/content/docs/contributors/custom-scheduler.md [hunk 3]
```diff
@@ -46,48 +46,26 @@ interfaces:
...
 
 ## Trying Out Your Scheduler
 
-Once you've implemented a custom configuration loader, you'll need to specify
-your loader by class using the `--config-loader` flag. If your loader relies on
-a configuration file, specify the path of that file using the `--config-file`
-flag. Here's an example [topology
+Once you've implemented a custom scheduler (or other components), you'll need to specify implementation of all interfaces above
```
Comment (billonahill) [inline website/content/docs/contributors/custom-scheduler.md]: s/implementation of all/the implementations of the

File: website/content/docs/contributors/custom-scheduler.md (hunk 1 above)
Comment (maosongfu) [inline website/content/docs/contributors/custom-scheduler.md]: sounds good; updated.

File: website/content/docs/contributors/custom-scheduler.md (hunk 2 above)
Comment (maosongfu) [inline website/content/docs/contributors/custom-scheduler.md]: sounds good; updated.

File: website/content/docs/contributors/custom-scheduler.md (hunk 3 above)
Comment (maosongfu) [inline website/content/docs/contributors/custom-scheduler.md]: sounds good; updated.

**Comment by maosongfu** [comment]
Rebased with master branch. Will open a new pull request to solve the commit tag conflicts.

**Pull request closed without merging**